PROTO_tests/tests/client_dtypes_test.py

```diff
@@ -0,0 +1,161 @@
+import random
+
+import pytest
+
+import arkouda as ak
+from arkouda import client_dtypes
+
+
+class TestClientDTypeTests:
+    """
+    Note: BitVector operations are not tested here because the class is
+    only a wrapper on a pdarray to display as such.
+    The class does not actually store values as bit-values,
+    it only converts to a bit representation for display.
+    Thus, pdarray testing covers these operations.
+    """
+
+    def test_BitVector_creation(self):
```

Comment:
could we parameterize this on `[ak.int64, ak.uint64]` to avoid some of the dupication?

Reply:
Updated